### PR TITLE
Generate keys commands

### PR DESCRIPTION
**Pull request by fbourigault**
Here is my proposal to implement #106.

The first commit is an import of the command and the test from `LexikJWTAuthenticationBundle` to make review of change easier.

# To do

- [x] Make `$algorithm` a command option as `LeagueOAuth2ServerBundle` doesn't have such configuration option.
- [x] Register the command as a service.

### CODE REVIEW

**Comment by fbourigault** [comment]
Can someone approve the workflows so I can get feedback from the whole matrix?

**Comment by chalasr** [comment]
My apologies for the delay Fabien, my OSS time was very limited lately I totally forgot to get back on this PR.
Could you please rebase?

**Comment by stefantalen** [comment]
ping @fbourigault @chalasr 

I think this still is a valuable addition, any updates on this?

File: composer.json [hunk 1]
```diff
@@ -17,12 +17,14 @@
...
     "require": {
         "php": "^8.1",
+        "ext-openssl": "*",
         "doctrine/doctrine-bundle": "^2.8.0",
         "doctrine/orm": "^2.14|^3.0",
         "league/oauth2-server": "^8.3",
         "nyholm/psr7": "^1.4",
         "psr/http-factory": "^1.0",
         "symfony/event-dispatcher": "^5.4|^6.2|^7.0",
+        "symfony/filesystem": "^5.3|^6.0",
```
Comment (fbourigault) [inline composer.json:27]: I need to update this.

**Comment by fbourigault** [comment]
I've rebased this PR and applied changes from LexikJWTAuthenticationBundle.

**Comment by chalasr** [comment]
@fbourigault Apologies for the delay here. I'm wiling to move forward :) Can you please have a look at failing tests?

**Comment by chalasr** [comment]
It took time, but here we go, this is in now. Thank you very much @fbourigault.

**Pull request merged**